modules/gdnative/arvr/arvr_interface_gdnative.cpp

```diff
@@ -370,16 +371,16 @@ void GDAPI godot_arvr_remove_controller(godot_int p_controller_id) {
 
 		// remove our tracker from our server
 		arvr_server->remove_tracker(remove_tracker);
-		memdelete(remove_tracker);
+		remove_tracker.unref();
```

Comment:
I think this is not actually needed, it will be cleared out when `remove_tracker` becomes out of scope